src/main/java/com/github/checkstyle/regression/module/ModuleUtils.java

```diff
@@ -0,0 +1,172 @@
+        }
+        else {
+            returnValue = false;
+        }
+        return returnValue;
+    }
+
+    /**
+     * Checks whether the corresponding file of a change may be considered as
+     * a Java main source file.
+     * @param change change to check
+     * @return true if the corresponding file of a change may be considered as
+     *      a Java main source file.
+     */
+    private static boolean isJavaMainSource(GitChange change) {
+        final boolean returnValue;
+        final Matcher matcher = JAVA_SOURCE_PARTTEN.matcher(change.getPath());
+        if (matcher.find()) {
+            returnValue = "main".equals(matcher.group(1));
+        }
+        else {
+            returnValue = false;
+        }
+        return returnValue;
+    }
+
+    /**
+     * Checks whether the corresponding file of a change may be considered as
+     * a test of checkstyle module.
+     * @param change change to check
+     * @return true if the corresponding file of a change may be considered as
+     *      a test of checkstyle module
+     */
+    public static boolean isCheckstyleModuleTest(GitChange change) {
+        final boolean returnValue;
+        if (JAVA_SOURCE_PARTTEN.matcher(change.getPath()).find()) {
+            final String fullName = convertJavaSourcePathToFullName(change.getPath());
+            if (fullName.endsWith(TEST_POSTFIX)) {
+                returnValue = NAME_TO_MODULE_EXTRAT_INFO.containsKey(
+                        fullName.substring(0, fullName.length() - TEST_POSTFIX.length()));
+            }
+            else {
+                returnValue = false;
+            }
+        }
+        else {
+            returnValue = false;
+        }
+        return returnValue;
+    }
+
+    /**
+     * Converts a {@link GitChange} of checkstyle module to a {@link ModuleExtractInfo}.
+     * @param change the change to convert
+     * @return the module info converted from the given change
+     */
+    public static ModuleExtractInfo convertModuleChangeToExtractInfo(GitChange change) {
+        final String fullName = convertJavaSourcePathToFullName(change.getPath());
+        return NAME_TO_MODULE_EXTRAT_INFO.get(fullName);
+    }
+
+    /**
+     * Converts a {@link GitChange} of test to the full name of corresponding checkstyle module.
+     * @param change the change to convert
+     * @return the full name of corresponding checkstyle module
+     */
+    public static String convertModuleTestChangeToModuleFullName(GitChange change) {
+        final String testName = convertJavaSourcePathToFullName(change.getPath());
+        return testName.substring(0, testName.length() - TEST_POSTFIX.length());
+    }
+
+    /**
+     * Converts a path of Java source file to its full qualified name.
+     * @param path the path of Java source file
+     * @return the corresponding full qualified name
+     */
+    private static String convertJavaSourcePathToFullName(String path) {
```

Comment:
`convertModuleChangeToExtractInfo` and `convertModuleTestChangeToModuleFullName` don't really need `GitChange` but you leave `convertJavaSourcePathToFullName` as `String` even though everywhere that uses it has `GitChange` available.

Change this parameter to GitChange.